docs/plugins.md

```diff
@@ -40,6 +40,10 @@ A plugin configuration can have attributes:
 
 By default, the logger plugin is enabled and configured to use the service `winston` (refer to the kuzzle-plugin-logger documentation for more information).  
 
+## Basic Auth
```

Comment:
To me, Basic auth is the one used by HTTP, i.e. [.htpasswd-like](https://en.wikipedia.org/wiki/Basic_access_authentication).
Could we rename it to something more generic like `LoginPassword` or `Password`?